src/libstore/s3-binary-cache-store.cc

```diff
@@ -169,6 +172,7 @@ struct S3BinaryCacheStoreImpl : public S3BinaryCacheStore
 {
     const Setting<std::string> profile{this, "", "profile", "The name of the AWS configuration profile to use."};
     const Setting<std::string> region{this, Aws::Region::US_EAST_1, "region", {"aws-region"}};
+    const Setting<std::string> scheme{this, "", "scheme", "The scheme to use for S3, http can generally be used since caches should not contain sensitive information."};
```

Comment:
I'd rather not generally advertise `http` as being a good choice.

Reply:
It's not enabled by default, it's just there for debugging, etc.